common/helpers/move/get-assessments.test.js

```diff
@@ -162,237 +162,237 @@ describe('Move helpers', function () {
           answers: [],
         })
       })
+    })
```

Comment:
With whitespace ignored, this comes up as a single change - just checking that you want to close off the `by default` block here

Reply:
Yes, I realised that the indentation was originally wrong. The `by default` should be separate to the `with Youth Risk Assessment`.